app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.js

```diff
@@ -62,17 +119,105 @@ function setConfigureError(err) {
   });
 }
 
+export function enableGroupTrigger() {
+  const namespace = NamespaceStore.getState().selectedNamespace;
+  const pipelineTriggers = PipelineTriggersStore.getState().triggers;
+  const configMappings = pipelineTriggers.triggersGroupRunArgsToAdd;
+  let runArgsMapping = {
+    arguments: [],
+    pluginProperties: [],
+  };
+  let propertiesConfigToEnable = {};
+  configMappings.arguments.map((arg) => runArgsMapping.arguments.push(arg.arg));
+  configMappings.pluginProperties.map((plugin) =>
+    runArgsMapping.pluginProperties.push(plugin.plugin)
+  );
+  configMappings.propertiesConfig.map(
+    (property) =>
+      (propertiesConfigToEnable = { ...propertiesConfigToEnable, ...property.propertiesConfig })
+  );
+  const triggerName = pipelineTriggers.triggersGroupToAdd.map(
+    (trigger) => trigger.programId.application
+  );
+  const scheduleName = `${pipelineTriggers.pipelineName}.${namespace}.${triggerName.toString()}`;
+
+  const requestObj = {
+    name: scheduleName,
+    description: '',
+    program: {
+      programName: pipelineTriggers.workflowName,
+      programType: 'WORKFLOW',
+    },
+    properties: Object.assign(
+      {},
+      {
+        'triggering.properties.mapping': JSON.stringify(runArgsMapping),
+        ...propertiesConfigToEnable,
+      }
+    ),
+    trigger: {
+      triggers: pipelineTriggers.triggersGroupToAdd,
+      type: pipelineTriggers.selectedTriggersType || PipelineTriggersTypes.orType,
+    },
+    constraints: [
+      {
+        maxConcurrency: 3,
+        type: 'CONCURRENCY',
+        waitUntilMet: false,
+      },
+    ],
+    timeoutMillis: 86400000,
```

Comment:
what is this timeout? if its second * hour * millisecond or something like that just do the 60 * 24 * 1000 or add a comment with that

Reply:
changed to 24 * 60 * 60 * 1000